### PR TITLE
feat(shim): extend json args

### PR DESCRIPTION
**Pull request by tknickman**

### CODE REVIEW

File: crates/turborepo-lib/src/shim.rs [hunk 1]
```diff
@@ -17,6 +17,9 @@ use turbo_updater::check_for_updates;
 use crate::{cli, get_version, PackageManager, Payload};
 
 static TURBO_JSON: &str = "turbo.json";
+// all arguments that result in pure json output to stdout
+static TURBO_JSON_OUTPUT_ARGS: [&str; 5] =
+    ["--json", "json", "--dry=json", "--graph", "--dry-run=json"];
```
[the remark this answers is not in the document]
Comment (NicholasLYang) [reply crates/turborepo-lib/src/shim.rs:22]: Maybe we should just not print the update notifier on dry runs. It overfits but at least it overfits in a consistent manner.

[the remark this answers is not in the document]
Comment (tknickman) [reply crates/turborepo-lib/src/shim.rs:22]: I like that - then we can always make it better and more precise later on

[the remark this answers is not in the document]
Comment (tknickman) [reply crates/turborepo-lib/src/shim.rs]: Ready for another look @NicholasLYang

**Review by NicholasLYang — APPROVED** [review]

**Comment by NicholasLYang** [comment]
CI's failing due to a typo but past that LGTM

**Pull request merged**